torch/csrc/jit/passes/python_print.cpp

```diff
@@ -1238,8 +1247,13 @@ struct PythonPrintPass {
   }
 
   void print(std::ostream& out, SourceRangeRecords& source_ranges_out) {
-    out << getImports() << body_.str();
-    source_ranges_out = body_.ranges();
+    out << getImports();
+    int64_t source_offset = out.tellp();
+    out << body_.str();
+    for (const auto& x : body_.ranges()) {
```

Comment:
I know this is small but this repeating the internal details of source range tracking outside of the the TaggedStream object. This will be a nasty and hard-to-find bug if someone changes the representation and doesn't realize the logic is also repeated here.